resources/js/components/catalog-new/NewYearSlider.vue

```diff
@@ -52,6 +80,12 @@ export default {
     components: {
         slider: VueSlider,
     },
+    methods: {
+        handleReset() {
+            this.value = { from: this.min, to: this.max }
+            this.$emit('reset')
```

Comment:
Couldn't we do something like this?
```suggestion
            this.$emit('change' { from: null, to: null })
```